package.json

```diff
@@ -170,7 +170,9 @@
     "@types/mock-fs": "^4.10.0",
     "@types/node": "^12.12.45",
     "@types/proper-lockfile": "^4.1.1",
+    "@types/react-beautiful-dnd": "^13.0.0",
     "@types/tar": "^4.0.3",
+    "array-move": "^3.0.0",
```

Comment:
I'd like to avoid adding random small utils like this. Lodash doesn't support smth similar?
Otherwise I sure it's very easy to replicate and put to out `common/utils`

Reply:
Lodash has specifically rejected the idea (https://github.com/lodash/lodash/issues/1701) and I got this package from that issue (https://github.com/lodash/lodash/issues/1701#issuecomment-621068361)